CodingPractice/PythonAssignments/tests/cleancode/test_CountItemsInLIst.py

```diff
@@ -0,0 +1,16 @@
+import unittest
+
+from CodingPractice.PythonAssignments.cleancode.CountItemsInList import count_items_in_list
+
+
+class CountItemsInListTests(unittest.TestCase):
+
+    def test_return_zero_for_empty_list(self):
+        self.assertEqual(0, count_items_in_list([]))
+
+    def test_non_empty_list(self):
+        self.assertEqual(5, count_items_in_list([0, 1, 2, 3, 4, 5]))
```

Comment:
Your test is failing here not only because of the count variable being changed inside the for loop, but also due to you appearing to be checking the index, rather than the number of variables in the test.